docs/css/custom.css

```diff
@@ -1112,11 +1112,13 @@ table {
   width: 100%;
   overflow-wrap: normal;
   border-collapse: collapse; /* Ensures that the borders collapse into a single border */
+  word-break: normal;
 }
 
 table th, table td {
   border: 1px solid #cccccc; /* Adds a border to each table header and data cell */
   padding: 6px 13px; /* Optional: Adds padding inside each cell for better readability */
+  max-width: 895px; /* Adjusted to `max-width(914px) of .content - padding` to avoid hidden content from a single cell */
```

Comment:
Hmm, so without this some table cells hide their content?

Reply:
No.